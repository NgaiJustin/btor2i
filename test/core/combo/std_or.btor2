; BTOR description generated by Yosys 0.33+65 (git sha1 90124dce5, clang 15.0.0 -fPIC -Os) for module std_or.
; ARGS: left=0b01010101010101010101010101010101 right=0b10101010101010101010101010101010
1 sort bitvec 32
2 input 1 left ; core.sv:107.35-107.39
3 input 1 right ; core.sv:108.35-108.40
4 or 1 2 3
5 output 4 out ; core.sv:109.35-109.38
; end of yosys output
