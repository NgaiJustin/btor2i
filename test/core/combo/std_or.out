
Environment:
0 : _
1 : _
2 : 2863311530
3 : 1431655765
4 : 4294967295
5 : _

out: [1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1]
